egs2/jsut/asr1/run.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+# Set bash to 'debug' mode, it will exit on :
+# -e 'error', -u 'undefined variable', -o ... 'error in pipeline', -x 'print commands',
+set -e
+set -u
+set -o pipefail
+
+fs=48000
+
+train_set=tr_no_dev
+dev_set=dev
+eval_set=eval1
+
+asr_config=conf/tuning/train_rnn.yaml
+decode_config=conf/tuning/decode_rnn.yaml
+lm_config=conf/lm.yaml
+
+./asr.sh \
+    --token_type char \
```

Comment:
Would you add the following?

```bash
fs=48000
if [ "${fs}" -eq 48000 ]; then
    # To suppress recreation, specify wav format
    opts=--audio_format wav --fs ${fs} --local_data_opts "--fs ${fs}"
else
    opts=--audio_format flac --fs ${fs} --local_data_opts "--fs ${fs}"
fi
```

You don't need to change the window length and frame shift parameter according to sampling rate for feats_type=raw for now.
I'll add it somehow by another way.